salesapp.js

```diff
@@ -29,7 +29,7 @@ function handleSubmit(event){
   console.log(formStore);
   console.log('User pressed submit button on form');
   console.log(storeArray);
-  formStoreArray.push(formStore);
+  formStoreArray.splice(formStore, 5);
```

Comment:
[Splice is an awesome method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)! I think the arguments here might be a little off though--the first argument should be an index in the array, and the second argument is the number of things you want to delete. In this case, it'll try to delete 5 things from an invalid index. The third argument and beyond is where you put in the things you want to add to the array.